src/components/Swap/SwapRoute.tsx

```diff
@@ -1,48 +1,45 @@
 import { Trade } from '@uniswap/sdk';
-import { Fragment, memo } from 'react';
+import React from 'react';
 import { ChevronRight } from 'react-feather';
 import { Box, Typography } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
 import { CurrencyLogo } from 'components';
 
-const useStyles = makeStyles(({ palette, breakpoints }) => ({
+const useStyles = makeStyles(({ palette }) => ({
   swapRoute: {
     padding: 12,
     border: `1px solid ${palette.divider}`,
     borderRadius: 12,
     display: 'flex',
     flexWrap: 'wrap',
-    justifyContent: 'space-evenly',
     alignItems: 'center',
-    '& > div': {
-      display: 'flex',
-      alignItems: 'center',
-      margin: '8px 0',
-      '& p': {
-        fontSize: 16,
-        lineHeight: '24px',
-        marginLeft: 6,
-      },
+    justifyContent: 'center',
+    '& p': {
+      fontSize: 16,
+      lineHeight: '24px',
+      marginLeft: 6,
     },
   },
 }));
 
-export default memo(function SwapRoute({ trade }: { trade: Trade }) {
+const SwapRoute: React.FC<{ trade: Trade }> = ({ trade }) => {
   const classes = useStyles();
   return (
     <Box className={classes.swapRoute}>
       {trade.route.path.map((token, i, path) => {
         const isLastItem: boolean = i === path.length - 1;
         return (
-          <Fragment key={i}>
-            <Box>
-              <CurrencyLogo currency={token} size='1.5rem' />
+          <Box key={i} display='flex' alignItems='center' my={0.5}>
+            <Box display='flex' alignItems='center'>
+              <CurrencyLogo currency={token} size='24px' />
               <Typography>{token.symbol}</Typography>
             </Box>
-            {isLastItem ? null : <ChevronRight />}
-          </Fragment>
+            {!isLastItem && <ChevronRight />}
```

Comment:
This code looks like a hack. Should probably be explained or written in a proper form

Reply:
Put a comment for this part and created a commit